vocab/credentials/credentials.jsonld

```diff
@@ -11,6 +11,8 @@
       "odrl": "http://www.w3.org/ns/odrl/2/",
       "xsd": "http://www.w3.org/2001/XMLSchema#",
       "JsonSchemaValidator2018": "cred:JsonSchemaValidator2018",
+      "JsonSchema2023": "cred:JsonSchema2023",
+      "CredentialSchema2023": "cred:CredentialSchema2023",
```

Comment:
```suggestion
```

This is modifying the v1 JSON-LD vocabulary, which we shouldn't be doing for something intended for v2.